domains/theo.json

```
@@ -5,6 +5,9 @@
     "discord": "691552610519613440"
   },
   "records": {
+    "MX": ["mx1.improvmx.com", "mx2.improvmx.com"],
+    "TXT": ["v=spf1 include:spf.improvmx.com ~all"],
```

since its a single TXT records and for readability
```suggestion
    "TXT": "v=spf1 include:spf.improvmx.com ~all",
```

---

alright-y, thanks!